scripts/parse_repo_json.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import json
+
+
```

Comment:
You might want to consider running this file through https://pypi.org/project/pycodestyle/